policyengine_canada/parameters/gov/provinces/nt/tax/income/credits/age_credit/max_amount.yaml

```diff
@@ -0,0 +1,13 @@
+description: Northwest Territories provides this maximum amount under the age credit.
+values:
+  # Amount increases with inflation every year
+  2022-01-01: 7_635
+  2023-01-01: 8_116
+metadata:
+  unit: currency-CAD
+  label: Northwest Territories age credit maximum amount
+  reference:
+  - title: 2022 Worksheet NT428 - line 58080
```

Comment:
need legal reference and add comment:
# Legal code amount does not adjust for inflation